model/jpa/src/main/resources/META-INF/jpa-changelog-18.0.0.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+  ~ * and other contributors as indicated by the @author tags.
+  ~ *
+  ~ * Licensed under the Apache License, Version 2.0 (the "License");
+  ~ * you may not use this file except in compliance with the License.
+  ~ * You may obtain a copy of the License at
+  ~ *
+  ~ * http://www.apache.org/licenses/LICENSE-2.0
+  ~ *
+  ~ * Unless required by applicable law or agreed to in writing, software
+  ~ * distributed under the License is distributed on an "AS IS" BASIS,
+  ~ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ * See the License for the specific language governing permissions and
+  ~ * limitations under the License.
+  -->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="keycloak" id="18.0.0-KEYCLOAK-10625-IDX_ADMIN_EVENT_TIME">
+        <!-- improve loading time of admin event list -->
+        <createIndex tableName="ADMIN_EVENT_ENTITY" indexName="IDX_ADMIN_EVENT_TIME">
+            <column name="ADMIN_EVENT_TIME" type="DESC"/>
```

Comment:
This index only indexes the column ADMIN_EVENT_TIME, and when I compare it with for example EVENT_ENTITY a similar index indexes also the REALM_ID. 

```
create or replace index IDX_EVENT_TIME
    on keycloak.EVENT_ENTITY (REALM_ID, EVENT_TIME);
``` 

Did you consider adding the REALM_ID as a first column to the index like for the other table?

Reply:
You're right, it makes totally sense to add the realm as well, I added it.
I also changed the index to make it as similar as possible to the index in `IDX_EVENT_TIME`